earthpy/tests/test__stack_bands.py

```diff
@@ -0,0 +1,31 @@
+""" Tests for the _stack_bands() method """
+
+import numpy as np
```

Comment:
do we need all of these modules? I'm guessing we need
* numpy
* pytest
* rasterio
* earthpy.spatial

Reply:
numpy is needed for the fixture. earthpy.spatial is required for the call to `_stack_bands`. probably don't need the others though.